netdisco/discoverables/harmony.py

```diff
@@ -0,0 +1,19 @@
+"""Discover Netgear routers."""
+from netdisco.util import urlparse
+from . import SSDPDiscoverable
+
+
+class Discoverable(SSDPDiscoverable):
+    """Add support for discovering Harmony Hub remotes"""
+
+    def info_from_entry(self, entry):
+        """Return the most important info from a uPnP entry."""
+        url = urlparse(entry.values['location'])
+        return (entry.description['device']['friendlyName'], url.hostname)
```

Comment:
We are in the process of upgrading all discovery formats. Can you please return a dictionary instead of a tuple.